src/CKAN.jl

```diff
@@ -1,23 +1,20 @@
 struct CKAN <: DataRepo
 end
 
-function description(repo::CKAN, mainpage)
-    desc = mainpage["notes"]
+description(repo::CKAN, mainpage) = mainpage["notes"]
+
+function author(repo::CKAN, mainpage)
     authors = [mainpage["author"]]
-    author = format_authors(authors)
-    maintainer = mainpage["maintainer"]
-    license = mainpage["license_title"]
+    format_authors(authors)
+end
+
+maintainer(repo::CKAN, mainpage) = mainpage["maintainer"]
+
+license(repo::CKAN, mainpage) = mainpage["license_title"]
+
+function createDate(repo::CKAN, mainpage)
     rawdate = Dates.Date(mainpage["metadata_created"][1:10], "yyyy-mm-dd")
-    date = Dates.format(rawdate, "U d, yyyy")
-    
-    """
-    Author: $(author)
-    License: $(license)
-    Date: $(date)
-    Maintainer: $(maintainer)
-
-    $(desc)
-    """
+    Dates.format(rawdate, "U d, yyyy")
```

Comment:
Do you thing we could do more date processing at at the metadata level?
We could allow  the Metadata to contain `Union{Missing, Dates.Date,  String}`
where the String is often just a year or some such.

then define:
```
format_dates(rawdate) = rawdate#Catchs the missing and the string
format_dates(rawdate::Dates.Date) = Dates.format(rawdate, "U d, yyyy")
```

Which would I think eliminate this line from most of the generators.
More D.R.Y

And it would later easily let us change the format for everything, since it would be in just one place